msw/pd.nsi

```diff
@@ -12,10 +12,16 @@
 ; Script generated by the HM NIS Edit Script Wizard.
 ; Manually edited by Roman Haefeli
 ; the string PDVERSION should be replaced by, e.g., 0.46-7
-; and the string WISHNAME by, e.g., wish86.exe  .
+; and the string WISHNAME by, e.g., wish86.exe.
 
 ; HM NIS Edit Wizard helper defines
-!define PRODUCT_NAME "Pure Data (${ARCHI}-bit)"
+
+; App name
+!if ${ARCHI} == "32"
+    !define PRODUCT_NAME "Pure Data (32-bit)"
+!else
+    !define PRODUCT_NAME "Pure Data"
+!endif
```

Comment:
sorry to be spuer-nitpicky, but wouldn't this make more sense (conceptually):

```
!if ${ARCHI} == "64"
    !define PRODUCT_NAME "Pure Data"
!else
    !define PRODUCT_NAME "Pure Data (${ARCHI}-bit)"
!endif
```

This way, the 8-bit version of Pd wouldn't be installed as "Pure Data" :yum: